Jenkinsfile

```diff
@@ -115,6 +115,45 @@ pipeline {
 				}
 			}
 		}
+		stage('Decide run load test') {
+			agent none
+			when {
+				environment name: 'DEPLOY_TO_TEST_SERVERS', value: 'yes'
+			}
+			steps {
+				slackSend color: 'good', channel: '#devops', message: "Server build ${env.BUILD_NUMBER} ready to start load test"
+				script {
+					env.RUN_LOAD_TEST = input message: 'User input required',
+					submitter: 'authenticated',
+					parameters: [choice(name: 'Run load test', choices: 'no\nyes', description: 'Choose "yes" if you want to run the load test')]
+				}
+			}
+		}
+		stage('Run load test') {
```

Comment:
So for my understanding, this step doesn't run with the load test server created in the stage before, but it runs on a 'yona' build agent?